app/(playground)/p/[agentId]/beta-proto/connector/utils.ts

```diff
@@ -0,0 +1,31 @@
+import type { GiselleNodeArchetype } from "../giselle-node/blueprints";
+import type { GiselleNodeCategory, GiselleNodeId } from "../giselle-node/types";
+import { createConnectorId } from "./factory";
+import type { ConnectorObject } from "./types";
+
+interface BuildConnectorArgs {
+	sourceNode: {
+		id: GiselleNodeId;
+		category: GiselleNodeCategory;
+		archetype: GiselleNodeArchetype;
```

Comment:
nits: I felt a little odd with the inconsistent capitalization of "ArcheType" and "archetype."

Reply:
thanks! We will proceed to manage and respond to https://github.com/route06inc/giselle/issues/56.